staging/src/k8s.io/apiserver/pkg/apis/apiserver/v1alpha1/types.go

```diff
@@ -167,3 +167,52 @@ type TracingConfiguration struct {
 	// Defaults to 0.
 	SamplingRatePerMillion *int32 `json:"samplingRatePerMillion,omitempty" protobuf:"varint,2,opt,name=samplingRatePerMillion"`
 }
+
+// +k8s:deepcopy-gen:interfaces=k8s.io/apimachinery/pkg/runtime.Object
+
+// PriorityAndFairnessConfiguration holds API Priority and Fairness parameters.
+type PriorityAndFairnessConfiguration struct {
+	metav1.TypeMeta `json:",inline"`
+
+	// WorkEstimator holds work estimator parameters.
+	// +optional
+	WorkEstimator WorkEstimatorConfiguration `json:"workEstimator,omitempty"`
+}
+
+// WorkEstimatorConfiguration holds work estimator parameters.
+type WorkEstimatorConfiguration struct {
+	// ListWorkEstimator holds work estimator parameters related to list requests.
+	// +optional
+	ListWorkEstimator ListWorkEstimatorConfiguration `json:"listRequests,omitempty"`
```

Comment:
Good, we got the word "configuration" out of this field name, but "work" and "estimator" remain.  The JSON field name also elides those words.  I think it would be great if the Go field name and the JSON field name had the usual relationship --- i.e., they are the same except for the initial case --- and used the shorter name.